jsx/Form.js

```diff
@@ -1795,7 +1799,7 @@ class RadioElement extends React.Component {
     let content = [];
     for (const key in this.props.options) {
       if (this.props.options.hasOwnProperty(key)) {
-        const checked = this.props.checked === this.props.items[key];
+        const checked = this.state.checked === this.props.options[key];
```

Comment:
so for example, this can just be 

`const checked = this.props.checked === this.props.options[key];`